tools/setup.py

```diff
@@ -26,10 +26,16 @@ def main():
 
 
 def write_lastchange():
-    run([
-        sys.executable, "build/util/lastchange.py", "-o",
-        "build/util/LASTCHANGE", "--source-dir", root_path, "--filter="
-    ])
+    with open("build/util/LASTCHANGE", "w+") as f:
+        f.write("LASTCHANGE=8ffa616f42c901c84b01441bb6902f213941fa85-\n")
```

Comment:
Does this work? I don't have this SHA in my git history.

Reply:
Another suggestion - this still unnecessary clobbered the file's mtime every time setup runs. Maybe you can check what's already in the file and avoid overwriting with the with the same data.